modules/web/src/app/shared/components/application-list/template.html

```diff
@@ -104,10 +104,10 @@
               <img *ngIf="getApplicationLogo(application.spec.applicationRef.name) as logo; else applicationDefName"
                    class="km-application-logo"
                    [src]="logo"
-                   [alt]="application.spec.applicationRef.name"
+                   [alt]="getApplicationName(application.spec.applicationRef.name)"
```

Comment:
Should we also use this in `matTooltip` below?

Reply:
I was thinking of leaving the tool tip to point to the actual name but can change it as well. WDYT?

Reply:
I think its fine if we leave it to `name`.